### PR TITLE
Setup horizontal funnel experiment

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Switch FF's to use experiment groups: 'control' and 'test'

Created a test on internal PH for this - https://app.posthog.com/experiments/24

https://github.com/PostHog/posthog/pull/8214#discussion_r790562359

## How did you test this code?

Locally overriding feature flags works

### CODE REVIEW

File: frontend/src/lib/constants.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const FEATURE_FLAGS = {
     FEEDBACK_CALL_CTA: 'feedback-call-cta', // owner: @paolodamico
     // Experiments / beta features
     INGESTION_GRID: 'ingestion-grid-exp-3', // owner: @kpthatsme
-    FUNNEL_HORIZONTAL_UI: '5730-funnel-horizontal-ui', // owner: @alexkim205
+    FUNNEL_HORIZONTAL_UI: '5730-funnel-horizontal-ui', // owner: @alexkim205 `control`, `test`
```
Comment (neilkakkar) [inline frontend/src/lib/constants.tsx:74]: Q: Did you delete the FF, so you could create the experiment with the same FF?

**Comment by neilkakkar** [comment]
Disabling the FF leads to a bit funky behaviour like this: 


I'd recommend not checking for `variant === 'control'`, and make that the default behaviour everywhere, so even if the FF doesn't load, we don't bork experience (or if you choose to run the experiment on a subset of participants)

File: frontend/src/lib/constants.tsx (hunk 1 above)
Comment (alexkim205) [reply frontend/src/lib/constants.tsx:74]: Yup I deleted the FF and recreated it with a new experiment. Is that the recommended way to use it? I didn't think I should create a separate FF for the same feature

**Comment by alexkim205** [comment]
> Disabling the FF leads to a bit funky behaviour like this:

good catch! fixing this now

File: frontend/src/lib/constants.tsx (hunk 1 above)
Comment (neilkakkar) [reply frontend/src/lib/constants.tsx:74]: Yep reasonable

**Review by neilkakkar — APPROVED** [review]

**Pull request merged**